cli/main.go

```diff
@@ -173,9 +172,33 @@ func getEnvInt(key string, defaultValue int) int {
 	return defaultValue
 }
 
-func parseRelayURLs(relayURLs string) []server.RelayEntry {
-	ret := []server.RelayEntry{}
-	for _, entry := range strings.Split(relayURLs, ",") {
+// Deletes whitespace from beginning and end, removes empty strings
+func trim(stringSlice []string) []string {
+	filtered := make([]string, 0, len(stringSlice))
```

Comment:
since the final size is not known, don't initialize the slice with a size?
```suggestion
	filtered := []string{}
```

Reply:
So that third argument is the capacity, not size. This is an optimization.

```go
func make([]T, len, cap) []T
```

Since we know the maximum number of items there can be (the length of the input slice), I think this is reasonable. Pre-allocations can speed things up quite a bit, not that this is a time sensitive operation.

Here's some random article that explains it pretty well:

* https://medium.com/swlh/bad-go-not-sizing-slices-aed1b01cff83